apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -66,6 +76,11 @@ abstract class TestAudioActivity extends Activity {
     public static final int ACTIVITY_TEST_DISCONNECT = 7;
     public static final int ACTIVITY_DATA_PATHS = 8;
 
+    public static final String KEY_DURATION = "duration";
```

Comment:
KEY and VALUE should be moved to IntentBasedTestSupport.java

Reply:
Done